apps/condo/domains/meter/tasks/sendSubmitMeterReadingsPushNotifications.js

```diff
@@ -0,0 +1,299 @@
+    // * Filter meters without readings
+    // * Filter meters without organisation default period
+    // * Join residents to rest of meters
+    // * Send message with specific unique key for set up readings (key is: {meterId}_{residentId}_{now().minusMonth().format(YYYY_MM)})
+    // * Filter meters with expired next verification date
+    // * Send message with specific unique key for expired verification (key is: {meterId}_{residentId}_{now().minusMonth().format(YYYY_MM)})
+    // * Print some stat
+
+    log('Start sending submit meter notifications', '=')
+    // initialize context stuff
+    const { keystone: context } = await getSchemaCtx('Meter')
+    const resourcesLocalizations = await loadAllLocalizedVersionsOfMeterResources({ context })
+
+    // let's load meters page by page
+    const state = {
+        pageSize: 100,
+        offset: 0,
+        hasMore: true,
+        processedMeters: 0,
+        processedPages: 0,
+        metersWithoutReadings: 0,
+        metersWithoutReadingsAndWithResidents: 0,
+        metersWithExpiredVerificationDate: 0,
+        startTime: dayjs(),
+    }
+    log(`Start proceeding meters page by page ${ state.startTime }`, '-')
+
+    do {
+        // log each 10 pages
+        if (state.processedPages % 10 === 0) {
+            log(`Processing page ${ state.processedPages + 1 }`)
+        }
+
+        // read all required data
+        const meters = await readMetersPage({
+            context,
+            offset: state.offset,
+            pageSize: state.pageSize,
+        })
+        const meterReadings = await readMeterReadings({ context, meters })
+        const organizations = await readOrganizations({ context, meters })
+
+        // right join data to metersPage
+        const metersWithoutReadings = rightJoin(
+            meters,
+            meterReadings,
+            (meter, item) => meter.id === item.meter.id,
+            (meter, readings) => ({ meter, readings })
+        )
+            .filter(({ readings }) => readings.length === 0)
+            .map(({ meter }) => meter)
+
+        // right join organizations
+        const metersWithoutPeriod = rightJoin(
+            metersWithoutReadings,
+            organizations,
+            (meter, organization) => meter.organization.id === organization.id,
+            (meter, organization) => {
+                /**
+                 * Detect message language
+                 * Use DEFAULT_LOCALE if organization.country is unknown
+                 * (not defined within @condo/domains/common/constants/countries)
+                 */
+                const lang = get(COUNTRIES, get(organization, 'country.locale'), DEFAULT_LOCALE)
+                const period = null // TODO calculate this prop after implementation submit period in organisation
+                return { ...meter, period, lang }
+            }
+        ).filter(({ period }) => period == null) // pick meters without organisation default period
```

Comment:
You need to use a consistent `code wrapping`: filter should be at the next lint like in 161 lint

Reply:
ok